doc/lua_api.txt

```diff
@@ -5380,20 +5380,49 @@ Definition tables
             place = <SimpleSoundSpec>,
             place_failed = <SimpleSoundSpec>,
         },
-        drop = "",
+
+        drop = "", --[[
         ^ Name of dropped node when dug. Default is the node itself.
-        ^ Alternatively:
+          Using a table allows multiple items, random selection among items,
+          palette control and tool filtering:
         drop = {
             max_items = 1,  -- Maximum number of items to drop.
             items = {  -- Choose max_items randomly from this list.
+                -- When using a shovel, they will get a drop about 60% of the time.
+                {
+                    -- Tool names are matched with `==`.
+                    tools = {
+                        "default:shovel_wood", "default:shovel_stone",
+                        "default:shovel_bronze", "default:shovel_steel",
+                        "default:shovel_mese", "default:shovel_diamond",
+                    },
+                    rarity = 2,  -- Probability of dropping is 1 / rarity.
```

Comment:
Is this a separate 1/rarity check for each possible item, as defined in max_items? Couldn't hurt to explain.